### ISSUE TITLE
Warning about discarding return value

### ISSUE THREAD

**Issue by tomdeakin** [illuhad/hipSYCL#221]
This HIP call expects the return value to to be discarded.
https://github.com/illuhad/hipSYCL/blob/5296ee524df046eaffa58ca4227f062a912bb4b6/include/hipSYCL/sycl/backend/clang.hpp#L57

```
/bin/../include/CL/../hipSYCL/sycl/backend/clang.hpp:57:3: warning: ignoring return value of function declared with 'nodiscard' attribute [-Wunused-result]
  hipConfigureCall(grid, block, shared, stream);
```

**Comment by illuhad** [comment]
... I think it expects it *not* to be discarded ;) The HIP headers are quite prone to producing warnings here and there, especially when compiling with clang.. We may have to add this warning to the list of warnings that are suppressed when including the HIP headers.

**Comment by tomdeakin** [comment]
That's what I meant. Note to self: proof read before sending.

**Comment by illuhad** [comment]
Should be fixed by PR #226.
